EIPS/eip-XXXX.md

```diff
@@ -0,0 +1,151 @@
+---
+eip: XXXX
+title: Datatokens as Token-Gated HTTPS Endpoints
+description: Composable RESTful and Solidity interface to implement token-gated HTTPS endpoints usind data tokens.
+author: Tim Daubenschütz (@TimDaub)
+discussions-to: https://tobedone
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-01
+requires: 712
+---
+
+## Abstract
+
+This standard introduces composable RESTful HTTPS and Solidity interfaces that e.g. enable paying to access HTTPS endpoints using [EIP-20](./eip-20.md) tokens.
+
+## Motivation
+
+According to RFC 7231, HTTPS status `402 Payment Required` is reserved for future use. But the future is now as Ethereum enables users to verify their identity with a server using public key infrastructure.
```

Comment:
Something about "the future is now" rubs me the wrong way...

Reply:
Yeah it's a bit too aggressive for a standard document. I'll make it more neutral.